addons/website_slides/data/mail_data.xml

```diff
@@ -63,6 +63,24 @@
             <field name="auto_delete" eval="True"/>
         </record>
 
+        <!-- Completed Channel Message -->
+        <record id="completed_channel_template" model="mail.template">
+            <field name="name">Completed Course</field>
+            <field name="model_id" ref="model_slide_channel"/>
+            <field name="subject">Congratulation ! You completed ${object.name}</field>
+            <field name="email_from">${(object.user_id.email_formatted or object.env.ref("base.user_admin").email_formatted) | safe}</field>
+            <field name="email_to">${user.email_formatted | safe}</field>
+            <field name="body_html" type="html">
+                <div style="margin: 0px; padding: 0px;">
+                    <div style="margin: 0px; padding: 0px; font-size: 13px;">
+                        <p>Congratulation <b>${user.name}</b>, you've completed the course ${object.name}</p>
```

Comment:
It is strange that current user is the target of the mail .... should probably be a template on slide.channel.partner so that you can use record.partner_id.name (for target) and record.channel_id.name (for course).